common/STcommon.py

```diff
@@ -1,10 +1,46 @@
 try:
 	import logging
+	import argparse
+	import configparser
 	from Crypto.PublicKey import RSA
+	from os.path import isfile
 except ImportError as e:
 	print("[-] {}, exiting".format(e))
 	exit(1)
 
+class Settings():
+	def __init__()
+		settings = configparser.ConfigParser()
+		settingsFile = ""
+		loaded = False
+
+	def createSettings(self,inFile='STcommon/settings.ini'):
+		self.settingsFile = inFile
```

Comment:
Should we add checks to see if we already have settings loaded? I am not sure what we would do if they were, but just a thought.